spec/feature_test_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'dish'
```

Comment:
Feature test (or integration test) files should be in their own folder 

e.g spec/features or spec/integration